samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,73 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+
+import click
+
+
+class BottleNecks:
+    def __init__(self, graph):
+        self.graph = graph
+
+    def ask_entry_point_question(self):
+        entry_points = self.graph.get_entry_points()
+
+        # All entry points must be calcualted before info can be displayed
+        while entry_points != []:
+            entry_point_question = (
+                "We found the following resources in your application that could be the entry point for a request."
+            )
+            item_number = 1
+            for item in entry_points:
+                item_name = item.get_name()
+                entry_point_question += "\n[%i] %s" % (item_number, item_name)
+                item_number += 1
+
+            entry_point_question += "\nWhere should the simulation start?"
+            user_input = ask(entry_point_question, 1, item_number - 1)
+
+            current_entry_point = entry_points.pop(user_input - 1)
+
+            self.ask_bottle_neck_questions(current_entry_point)
+
+            self.graph.add_resource_to_analyze(current_entry_point)
+
+        click.echo("Running calculations...")
+
+    def ask_bottle_neck_questions(self, resource):
+        if resource is None:
+            return
+
+        if resource.get_resource_type() == "AWS::Lambda::Function":
```

Comment:
Reference `samcli\commands\_utils\resources.py::AWS_LAMBDA_FUNCTION` instead of hardcoding it here.